client/me/help/help-contact/index.jsx

```diff
@@ -90,11 +90,15 @@ const HelpContact = React.createClass( {
 		olarkActions.hideBox();
 	},
 
-	componentDidUpdate: function() {
+	componentDidUpdate: function( prevProps ) {
 		// Directly initialization is a noop if it's already happened. This catches
 		// instances where a state/prop change moves a user to Directly support from
 		// some other variation.
 		this.prepareDirectlyWidget();
+
+		if ( ! prevProps.isDirectlyReady && this.props.isDirectlyReady ) {
+			analytics.tracks.recordEvent( 'calypso_help_contact_directly_initialize_success' );
```

Comment:
This is the old way of recording events :trollface: 

We should use the [analytics middleware](https://github.com/Automattic/wp-calypso/tree/master/client/state/analytics) and probably add this [here](https://github.com/Automattic/wp-calypso/blob/master/client/state/data-layer/third-party/directly/index.js#L23) as `.then( () => dispatch( recordEvent( 'calypso_help_directly_initialize_success' ) ) )`